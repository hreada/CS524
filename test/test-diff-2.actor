letactor(
  bind(dummy, lambda(m, m)),
  bind(diff, rec(lambda(beh, lambda(m,
    seq(send(dummy, minus(fst(m), snd(m))),
        become(beh)))))),
  send(diff, pr(42, 24)))
