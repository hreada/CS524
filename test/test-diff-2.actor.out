<T>
  <actors>
    <actor>
      <k> exec ( nil ) </k>
      <id> _13 </id>
    </actor>
    <actor>
      <k> ready ( lambda ( _8 , app ( app ( lambda ( _7 , app ( lambda ( beh , lambda ( m , seq ( send ( _1 , minus ( fst ( m ) , snd ( m ) ) ) , become ( beh ) ) ) ) , lambda ( _8 , app ( app ( _7 , _7 ) , _8 ) ) ) ) , lambda ( _7 , app ( lambda ( beh , lambda ( m , seq ( send ( _1 , minus ( fst ( m ) , snd ( m ) ) ) , become ( beh ) ) ) ) , lambda ( _8 , app ( app ( _7 , _7 ) , _8 ) ) ) ) ) , _8 ) ) ) </k>
      <id> _2 </id>
    </actor>
    <actor>
      <k> exec ( 18 ) </k>
      <id> _1 </id>
    </actor>
    <actor>
      <k> exec ( nil ) </k>
      <id> initactor </id>
    </actor>
  </actors>
  <messages> .MessageCellBag </messages>
  <definedaddr> ListItem ( initactor ) ListItem ( _1 ) ListItem ( _2 ) ListItem ( _13 ) </definedaddr>
</T>
